site/docs/extensions/custom-view.md

```diff
@@ -60,13 +60,8 @@ This extension adds the ability to create a custom view to display the data.
 
 ### Icons
 
-- Toggle custom view:
-    * Bootstrap3: `glyphicon glyphicon-eye-open`
-    * Bootstrap4: `fa fa-eye`
-    * Semantic: `fa fa-eye`
-    * Foundation: `fa fa-eye`
-    * Bulma: `fa fa-eye`
-    * Materialize: `remove_red_eye`
+- customViewOn: `glyphicon glyphicon-eye-open`
+- customViewOff: `glyphicon glyphicon-eye-open`
```

Comment:
@marceloverdijk, can you help update the Icons with different themes?

Reply:
Hi @wenzhixin , code wise the icons are updated for the different themes as requested by @UtechtDustin before. See this commit: https://github.com/wenzhixin/bootstrap-table/pull/6534/commits/1727905c431f5a4707b4be444f5928cae3a60cab

Or do you mean updating the docs? To mention per theme the used icons?
I removed that detail intentionally as I notices other docs also do not mention that.
But if you prefer I can add it again with the updated icons.

Or do you mean something different?

Reply:
Sure, I will update it later today 👍

Reply:
@wenzhixin I've pushed the update documenting the used icons.